woocommerce/woocommerce-gutenberg-products-block
# Critical flows: Merchant → Checkout → Can adjust T&S and Privacy Policy options

Partially fixes #5793

### Notes

This PR adds an e2e test to verify that a merchant can adjust the T&C and Privacy Policy links on the checkout block.

### Testing

#### Automated Tests

### Manual Testing

1. Check out this PR.
2. Start Docker Desktop.
3. Run `npm run wp-env destroy && npm run wp-env start` to start a fresh e2e environment.
4. Run `npm run test:e2e -- checkout-terms` to run only this test case.
5. Run `npm run test:e2e` to run the whole test suite.
